tools/Microsoft.AspNet.Server.Kestrel.GeneratedCode/KnownHeaders.cs

```diff
@@ -388,42 +388,49 @@ protected void CopyToFast(ref MemoryPoolIterator2 output)
                     if (_raw{header.Identifier} != null) 
                     {{
                         output.CopyFrom(_raw{header.Identifier}, 0, _raw{header.Identifier}.Length);
-                    }} else ")}
-                    foreach(var value in _{header.Identifier})
-                    {{
-                        if (value != null)
+                    }} 
+                    else ")}
+                        foreach (var value in _{header.Identifier})
                         {{
-                            output.CopyFrom(_headerBytes, {header.BytesOffset}, {header.BytesCount});
-                            output.CopyFromAscii(value);
+                            if (value != null)
+                            {{
+                                output.CopyFrom(_headerBytes, {header.BytesOffset}, {header.BytesCount});
+                                output.CopyFromAscii(value);
+                            }}
                         }}
-                    }}
                 }}
             ")}
         }}" : "")}
         public unsafe void Append(byte[] keyBytes, int keyOffset, int keyLength, string value)
         {{
-            fixed(byte* ptr = keyBytes) {{ var pUB = ptr + keyOffset; var pUL = (ulong*)pUB; var pUI = (uint*)pUB; var pUS = (ushort*)pUB;
-            switch(keyLength)
-            {{{Each(loop.HeadersByLength, byLength => $@"
-                case {byLength.Key}:
-                    {{{Each(byLength, header => $@"
-                        if ({header.EqualIgnoreCaseBytes()}) 
-                        {{
-                            if ({header.TestBit()})
+            fixed (byte* ptr = &keyBytes[keyOffset]) 
```

Comment:
Other than the line above (and `var pUB = ptr;`), are the rest of the changes just formatting?